polaris-react/src/components/InContextLearning/InContextLearning.tsx

```diff
@@ -71,6 +71,7 @@ export function InContextLearning({onDismiss, title}: Props) {
         activator={currentActivator}
         render={(state) => {
           const isAbove = state.positioning === 'above';
+          const horizontalMargin = {marginLeft: '.5rem', marginRight: '0.5rem'};
```

Comment:
I moved this out in a different PR - if you pull in the latest you can move it into the InContextLearning.scss!

Reply:
Taking care of it now.

Reply:
Done 👍